src/ApiCompat/Microsoft.DotNet.ApiCompat.Shared/SuppressionFileHelper.cs

```diff
@@ -25,14 +26,14 @@ internal static class SuppressionFileHelper
 
             if (suppressionOutputFile == null)
             {
-                throw new ArgumentException(CommonResources.SuppressionsFileNotSpecified, nameof(suppressionOutputFile));
+                log.LogError(CommonResources.SuppressionsFileNotSpecified);
+                return;
```

Comment:
Unrelated change. Before, we were throwing an exception when a suppression file path wasn't specified but the generate suppression file property/flag was set. Now, with our improved logging interface, we can actually emit an error without a suppression (which wasn't possible before).